changes/api/622.feature.md

```diff
@@ -0,0 +1,5 @@
+Added `VoidAction()` action to match the keysym pair `NoSymbol`/`VoidSymbol`.
+It enables erasing a previous action and breaks latches.
+
+This is a libxkbcommon extension. When serializing it will be converted to
+`LockControls(controls=none,affect=neither)` for backward compatibility.
```

Comment:
Does LockControls also break latches?

Reply:
[It does](https://gitlab.freedesktop.org/xorg/proto/xorgproto/-/blob/1914233e662d23ffb3812b80fadd0bbd064ad91c/include/X11/extensions/XKB.h#L508-L512). See also our own `xkb_action_breaks_latch()`.

Reply:
In that case, AIUI, VoidAction is just a nicer alias for the LockControls action. If so, this LGTM.

Reply:
Exactly. This brings better semantics and we can make it a proper independent action later. I chose `LockControls(controls=none,affect=neither)` because it seems the only portable action we support that would cause no side effect other than breaking latches. `SetControls()` may be fine too, but the `affect=neither` field of `LockControls` seems  to better ensure it is a dummy action.